linux_os/guide/system/permissions/restrictions/coredumps/coredump_disable_backtraces/rule.yml

```diff
@@ -21,6 +21,7 @@ identifiers:
 
 references:
     ospp: FMT_SMF_EXT.1
+    srg: SRG-OS-000480-GPOS-00227
```

Comment:
@redhatrises should this check be updated to say "if coredumps are enabled, then check this?"

Reply:
As this is part of systemd, I think that this should still be checked as systemd configuration files delivered by vendor could change.